src/components/Todo/Todo.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { User } from '../User';
+
+export const Todo = ({ user, todo }) => (
+  <>
+    <div className="todo">
+      <span className="todo__title">{todo.title}</span>
+      <span className={
+        `todo__is-completed todo__is-completed--${
+          todo.completed ? 'true' : 'false'
+        }`
```

Comment:
```suggestion
      <span className={
        `todo${
          todo.completed ? '--completed' : ''
        }`
```
u can do it such way